README.org

```diff
@@ -1009,6 +1009,9 @@ For additional git related emacs packages to use or to get inspiration from, tak
    - [[http://endlessparentheses.com/][Endless Parentheses]] - Concise ramblings on Emacs productivity.
    - [[http://whattheemacsd.com/][What the .emacs.d!?]] - Sharing Emacs snippets and pastes with others.
    - [[https://hackr.io/tutorials/learn-emacs][Community Curated Emacs Resources]] - Top Recommended Resources.
+   - [[https://github.com/p3r7/awesome-elisp][awesome-elisp]] - An awesome list for Elisp development.
+   - [[https://github.com/alphapapa/emacs-package-dev-handbook][emacs-package-dev-handbook]] - A handbook for package developers.
+   - [[https://emacs.zeef.com/ehartc][emacs-zeef]] - Another list linking to Emacs resources.
```

Comment:
emacs-zeef - it seems like some of the content sounds vaguely familiar to awesome-emacs itself 

For example, in Zeef "I tried to sort the items by its popularity (roughly, because it’s impossible to have a precious standard. e.g. most of people would use it; recommended for every newbie…)."

vs in awesome-emacs "Please don’t rearrange the package ordering without any reason! The items should be sorted by its popularity ( roughly, because it’s impossible to have a precise standard. e.g. most people would use it; recommended for every newbie…), instead of your personal preference."

Reply:
Yeah, it is similar but I thought the more resources the better. I'm okay with removing it though if you think it doesn't belong here.

Reply:
Yes, as to removing that link.

Reply:
Okay, done